ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/WorkloadCvesOverviewPage.tsx

```diff
@@ -236,6 +242,8 @@ function WorkloadCvesOverviewPage() {
         pagination.setPage(1);
         sort.setSortOption(getDefaultSortOption(entityTab), 'replace');
 
+        setDefaultSearchFilter(entityTab === 'CVE' ? 'Image CVE' : entityTab);
```

Comment:
Nit: I think it would be slightly preferable to explicitly call out the filter for each possible entity tab. Although the entity tab can have values `'Image'` or `'Deployment'`, it is somewhat coincidental that they match up exactly with the filter names.

Reply:
Yeah, that's a good point. The tabs might change in the future and it can potentially get misaligned